js/background.js

```diff
@@ -498,16 +505,24 @@
             }
 
             return wrapDeferred(conversation.save(updates)).then(function() {
-                if (typeof details.expireTimer !== 'undefined') {
-                    var source = textsecure.storage.user.getNumber();
-                    var receivedAt = Date.now();
-                    return conversation.updateExpirationTimer(
-                        details.expireTimer,
-                        source,
-                        receivedAt,
-                        {fromSync: true}
+                const { expireTimer } = details;
+                const isValidExpireTimer = typeof expireTimer === 'number';
+                if (!isValidExpireTimer) {
+                    console.log(
+                        'Ignore invalid expire timer.',
+                        'Expected numeric `expireTimer`, got:', expireTimer
                     );
+                    return;
                 }
+
+                var source = textsecure.storage.user.getNumber();
+                var receivedAt = Date.now();
+                return conversation.updateExpirationTimer(
+                    expireTimer,
+                    source,
+                    receivedAt,
+                    {fromSync: true}
+                );
```

Comment:
We should dedupe the two identical code snippets above but I didn’t to reduce risk for the hotfix.